custom_connectors/custom_auth/kizen.rb

```diff
@@ -56,6 +56,119 @@
       input
     end,
 
+    format_payload: lambda do |payload|
+      if payload.is_a?(Array)
+        payload.map do |array_value|
+          call('format_payload', array_value)
+        end
+      elsif payload.is_a?(Hash)
+        payload.map do |key, value|
+          key = call('inject_special_characters', key)
+          if value.is_a?(Array) || value.is_a?(Hash)
+            value = call('format_payload', value)
+          end
+          { key => value }
+        end.inject(:merge)
+      end
+    end,
+
+    format_schema: lambda do |schema|
+      if schema.is_a?(Array)
+        schema.map do |array_value|
+          call('format_schema', array_value)
+        end
+      elsif schema.is_a?(Hash)
+        schema.map do |key, value|
+          if %w[name].include?(key.to_s)
+            value = call('replace_special_characters', value.to_s)
+          elsif %w[properties toggle_field].include?(key.to_s)
+            value = call('format_schema', value)
+          end
+          { key => value }
+        end.inject(:merge)
+      end
+    end,
```

Comment:
We have done away with this method. This can be seen in the `object_input` method as standard input fields do not have special characters anyway.

Reply:
If I change these references to object_input, the object input method does not have the correct 'cases' for the other objects(activities, interactions, etc.).